### PR TITLE
Add roles to homepage

### LINKED ISSUE #23: Create homepage roles section
Screen Shot 2022-08-07 at 2 03 39 PM

### PR DESCRIPTION
**Pull request by harsohailB**
Resolves #23 

Screen Shot 2022-08-07 at 2 04 38 PM

cards

### CODE REVIEW

File: components/Carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import "swiper/css";
+import "swiper/css/effect-cards";
+
+const Carousel = ({ effect, grabCursor, modules, children }) => {
+  return (
+    <>
+      <Swiper effect={effect} grabCursor={grabCursor} modules={modules} className="mySwiper">
+        {children}
+      </Swiper>
+    </>
```
Comment (tylerlam1) [inline components/Carousel.jsx]: I don't think we need the fragments here right?

File: components/Carousel.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
```
Comment (tylerlam1) [inline components/Carousel.jsx]: Since we removed the fragments, I think we can remove this as well right?

File: components/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import { Swiper } from "swiper/react";
+import "swiper/css";
+import "swiper/css/effect-cards";
+
+const Carousel = ({ effect, grabCursor, modules, children }) => {
+  return (
+    <>
+      <Swiper effect={effect} grabCursor={grabCursor} modules={modules} className="mySwiper">
+        {children}
+      </Swiper>
```
Comment (tylerlam1) [inline components/Carousel.jsx]: We could smooth this out with the following:

```
const Carousel = ({children, ...props}) =. {
  return (
    <Swiper {...props} className="mySwiper">{children}</Swiper>
  )
}
```

File: pages/index.js [hunk 4]
```diff
@@ -1,3 +1,61 @@
...
+import Button from "../components/Button";
+import Carousel from "../components/Carousel";
+import Tile from "../components/Tile";
+
+const ROLE_TILE_SIZE = 300;
+const ROLES = ["Development", "Marketing", "Design"];
+
+const RoleTile = ({ role }) => {
+  return (
+    <div className="flex flex-col">
```
Comment (tylerlam1) [inline pages/index.js]: What's this outer `div` doing? It doesn't look like it's doing anything new to me, right?

File: pages/index.js [hunk 5]
```diff
@@ -1,3 +1,61 @@
...
 export default function Home() {
-  return <div className="flex justify-center"></div>;
+  return (
+    <div className="flex flex-col">
+      <div className="px-10 md:px-24 lg:px-48 py-20 bg-[#FF6B54]">
+        <h1 className="text-5xl font-semibold text-white">Ready to join the change?</h1>
+        <p className="text-xl text-white py-10 font-medium">
+          Explore our roles here and follow us on our socials to know when we’re recruiting! :)
+        </p>
+        <div className="hidden md:flex flex-col md:flex-row justify-between space-y-2 md:space-y-0 md:space-x-2">
```
Comment (tylerlam1) [inline pages/index.js:114]: Just wondering, how does this unhide - on mobile phones? I see it works, but wouldn't we need to do something like `md:visible`?

**Review by tylerlam1 — CHANGES_REQUESTED** [review]
Looks great, I have some suggestions though!

File: pages/index.js (hunk 5 above)
Comment (harsohailB) [reply pages/index.js:114]: * `hidden` makes it `display: none` on small screens
* `md:flex` makes it `display: flex` on large screens - `flex` is unhide basically

**Review by tylerlam1 — APPROVED** [review]
lgtm

**Pull request merged**